### DISCUSSION TITLE
Allow uploading files from other apps on Android via the Share mechanism

### DISCUSSION THREAD

**Discussion by ItsHarper** [standardnotes/forum#2832] [Suggestions]
I originally posted this [here](https://github.com/standardnotes/mobile/issues/632), but I'm reposting (with some more detail) since that repository has been archived.

---

Standard Notes should expose an entry on the standard Android share screen, which allows the user to attach a file to a note.

For an example use case, if I use the Pixel Recordings app to make a recording, I should be able to share the audio file and transcript directly to Standard Notes for upload, rather than having to first export the recording to general device storage **using a third party file manager**, upload it to Standard Notes, and finally delete the copy in general storage. If I didn't have a file manager app installed that adds a "Save as" option to the Share menu, I'd have had to export my recording by using an insecure cloud service!

This may apply to iOS too.

**Comment by myreli** [comment]
> I originally posted this https://github.com/standardnotes/mobile/issues/632, but I'm reposting (with some more detail) since that repository has been archived.

Thanks for bringing this up @NoahAndrews! I'm currently triaging older issues and migrating the forum to Discussions. Once I'm finished I'll review the other repositories.
